src/navigation/LoginStack.js

```diff
@@ -84,12 +92,16 @@ const LoginStack = createStackNavigator(
   },
   {
     initialRouteName: 'Start',
-    headerLayoutPreset: 'center',
+    headerLayoutPreset: 'left',
```

Comment:
Why left? Mb we should left "center"?

Reply:
This is temporary solution for long titles. In case of center it look like "Very long tit..." instead of "Very long title".